dev.js

```diff
@@ -34,7 +34,7 @@ var loadClientBundleLink = path.join(dirs.meteor, 'client/loadClientBundle.html'
 
 var requireServerBundleJs = path.join(dirs.meteor, 'server/require.server.bundle.js');
 
-require('./core-js-custom-build');
+exec('node core-js-custom-build.js');
 
```

Comment:
Ah right, makes sense.  Yeah I think `exec` is better than going to the trouble of passing a callback from the main scripts